spock-specs/src/test/groovy/org/spockframework/smoke/parameterization/DataTables.groovy

```diff
@@ -379,6 +546,16 @@ a | b | c
     4 | 5 || 9
   }
 
+  def "cells can be separated with any amount of semicolons"() {
+    expect:
+    a + b == c
+
+    where:
+    a ; b ;;   c
+    1 ; 2 ;;;  3
```

Comment:
I don't know if we can fix this or if the compiler throws it away already, but I don't really like this. IMHO every row should have the same identifiers

Reply:
Well, we also do not enforce this for pipe and double-pipe and there are even tests that make sure you can freely mix thme.
But anyway, as you assumed, the parser throws them out already and there is no way to differentiate this case as far as I have seen.